extensions/amp-date-picker/validator-amp-date-picker.protoascii

```diff
@@ -191,3 +191,69 @@ attr_lists: {
     value: ""
   }
 }
+
+# HTML5, 4.11.3 The template element
```

Comment:
The `template` tag is defined in the `amp-mustache` extension as well. Please add a comment here noting that and also there noting these are defined here. See https://github.com/ampproject/amphtml/blob/master/extensions/amp-mustache/validator-amp-mustache.protoascii